app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/RecFragmentTest.kt

```diff
@@ -43,11 +43,11 @@ class RecFragmentTest {
 
 
         // Check that the recording file was saved and has non-zero size
-        scenario.onFragment { fragment ->
-             val recordingFile = File(fragment.getOutputFilePath())
-             assertTrue(recordingFile.exists())
-             assertTrue(recordingFile.length() > 0)
-        }
+        // scenario.onFragment { fragment ->
```

Comment:
You can delete it if it's no more relevant

Reply:
It will probably be relevant once the whole workflow is in place again so that’s why I kept it.
We can also delete it and retrieve it back with git if necessary.